.github/workflows/prerelease-beta-patch.yml

```diff
@@ -0,0 +1,42 @@
+# (C) 2024 GoodData Corporation
+
+name: Prerelease ~ Publish beta for patch
+on:
+    workflow_dispatch:
+      inputs:
+        version:
+            required: true
+            description: "Specify the minor version to patch (example: 9.5.0)"
+            type: string
+jobs:
+    check-input-version-param:
+        runs-on: ubuntu-latest
+        outputs:
+            branch: ${{ steps.check-input-version-param.outputs.branch }}
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+            - name: Check if branch for version exists
+              id: check-input-version-param
+              run: |
+                  base="rel/${VERSION%.*}"
+                  # TODO: brackets for if?
```

Comment:
please unify the brackets or just remove TODO

Reply:
done